mobile/src/Student/screens/EnterGameCode/index.js

```diff
@@ -72,11 +67,18 @@ const EnterGameCode = ({ navigation, fetchGameSessionByCode }) => {
                             style={styles.enterButton}
                             onPress={this.onGameCodeSubmit}
                         />
+                        {showErrorText && <Text
+                            style={{ ...styles.errorText }}>
+                            We were unable to{"\n"}join this game.{"\n"}
+                            {"\n"}Check the Game Code{"\n"}and try again.
+                        </Text>
+                        }
```

Comment:
Would you mind checking this on a smaller size iPhone and see if this still fits the screen? You can do this by changing the iPhone name in the [package.json](https://github.com/rightoneducation/righton-app/blob/dev/mobile/package.json#L10) to "iPhone 13 mini" or any other mini simulator that's already installed. You can see the list in the Xcode

Reply:
@Alexandra-Kushnirsky, thanks for confirming the formatting with this image:
![image](https://user-images.githubusercontent.com/79417944/210879790-fc9a629c-27f9-40cf-8191-2698006e880f.png)

I am resolving this conversation and merging.